modules/internal/String.chpl

```diff
@@ -1743,6 +1743,16 @@ module String {
     }
   }
 
+  /*
+     :returns: A string with the single character with the ASCII value `i`.
+  */
+  inline proc asciiToString(i: uint(8)) {
+    var buffer: [0..1] uint(8);
+    buffer[0] = i;
+    buffer[1] = 0;
+    return new string(c_ptrTo(buffer), 1, 2);
```

Comment:
I would probably use chpl_here_alloc / free rather than using the array -> c_ptrTo mechanism, since things like multi-ddata make such uses of arrays problematic. I'd like to limit any problems in such areas to tests of extern functions.